_episodes/05c-sets-dictionaries.md

```diff
@@ -0,0 +1,193 @@
+---
+title: "Sets and Dictionaries"
+teaching: 10
+exercises: 5
+questions:
+- "What is a set?"
+- "What is a dictionary?"
+objectives:
+- "Explain the features of sets and dictionaries."
+- "Understand how dictionaries map keys to values"
+- "Learn how to add and get items from sets and dictionaries."
+keypoints:
+- "A set is an unordered, unique collection of **immutable** objects."
+- "A dictionary is a mapping between objects, from **keys** to **values**."
+- "Dictionary keys must be **immutable**."
+- "Get a value from a dictionary with `my_dict[some_key]`. Add an item with `my_dict[some_key] = some_value`."
+- "Dictionaries and sets are mutable."
+---
+
+# Sets
```

Comment:
Sets and dictionaries were already introduced in the iterable data type section. It seems like jumping around to suddenly come back to these data types now. Could this all be integrated into the iterable lesson to avoid confusion? It doesn't make sense to me to use sets in the earlier lesson and then explain them here. I think that should all go together.